docs/tool/GUI.ModsManager/FAQ.md

```diff
@@ -0,0 +1,55 @@
+# OpenKH Mod Manager FAQs
+
+### Return to the [Main Page](index.md)
+
+### Special Thanks to:
+* Shananas for the continued upkeep of the OpenKH Mod Manager when something new arises
+* MainMemory for her updating of Panacea to work with the PC releases
+* Xeeynamo for the initial rough version of the program
+* Alios for discovering a major issue with the Mod Manager so that I may include an answer to the problem in this FAQs document
+* Everyone else in the OpenKH community for helping make the modding environment for these games as welcoming and exciting as it has been
+
+## [Table of Contents]()
+* ### [Is extracting my games mandatory for the mod manager to work?](#q-is-extracting-my-games-mandatory-for-the-mod-manager-to-work)
+* ### [Lua mods aren't working!](#q-lua-mods-arent-working)
+* ### [Does OpenKH support Steam Deck/Linux?](#q-does-openkh-support-steam-decklinux)
+* ### [I moved my OpenKH install and now I can't use the mod manager!](#q-i-moved-my-openkh-install-and-now-i-cant-use-the-mod-manager)
+* ### [I can't open the mod manager on my removeable storage device!]()
+* ### [I want to start making mods but don't know how](#q-i-want-to-start-making-mods-but-dont-know-how)
+
+---
+
+### Q: Is extracting my games mandatory for the mod manager to work?
+
+> **A:** For the best user experience, yes. We understand it takes up a lot of storage, but it allows you to retain your unmodified game files through a dynamic file replacement system instead of relying on a slow and tedious patching method. Some tips to reduce the storage used by game extraction are:

```

Comment:
As I leave another i shoulve started a review but too late. to be completely truthful which im also not on the game extraction page. Any Lua only mod will work fine and any PCPatch will work fine. Basically any mod that just replaces an entire file outright will work for both panacea and patching. what doesnt work is mod manager functions beyond a simple copy. All binarcs, spawnpoint, listpatch, etc. Idk if you want to include any of this but the extract game data page for PC at least says Lua Scripts will work fine.